js/btnclick.js

```diff
@@ -0,0 +1,265 @@
+//all the dom elements
+let shapes = document.querySelector(".shapes");
+let next_btn = document.querySelector(".nxt_btn");
+const circleTick = document.querySelector(".tick");
+const triangleTick = document.querySelector(".triangle_tick");
+const squareTick = document.querySelector(".square_tick");
+const header = document.querySelector(".learning_header");
+const mainContent = document.querySelector(".learning_content");
+
+//object for storage.
+//provided if we add one more figure we need to only define the area of the figure and its respective perimeter formula
+let shape = {
+  name: "",
+
+  sideOrRadius: "",
+
+  isRadius: true,
+
+  radius: "",
+
+  base: "",
+
+  height: "",
+
+  //area specific to each figures
+  area: function () {
+    if (this.isRadius) {
+        //if this.isRadius will calculate the area of circle
+      return 3.14 * (this.radius * this.radius);
+    } else if (this.name === ".triangle") {
+        //else trianlge
+      return 0.433 * this.base * this.height;
+    } else {
+        //else square
+      return this.base * this.base;
+    }
+  },
+
+  //perimeter specific to each figures
+  perimeter: function () {
+    if (this.isRadius) {
```

Comment:
can u try switch statements

Reply:
switch statement has been used condition being taken on the shape name.